common/utils/openebs/upgrade/test.yml

```diff
@@ -0,0 +1,62 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+#        - include_tasks: /common/utils/application_liveness_check.yml
+#          when: liveness_label != ''
+
+        - include: test_prerequisites.yml
+
+         ## RECORD START OF THE TEST IN LITMUS RESULT CR
+        - include_tasks: /common/utils/create_testname.yml
+ 
+       ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: /common/utils/update_litmus_result_resource.yml
+          vars:
+            status: 'SOT'
+
+        # Ensuring if the application is running before starting upgrade.
+        - include_tasks: /common/utils/status_app_pod.yml
+          vars:
+             app_ns: "{{ namespace }}"
+             app_lkey: "{{ app_label.split('=')[0] }}"
+             app_lvalue: "{{ app_label.split('=')[1] }}"
+             delay: 10
+             retries: 5
+       
+       # TRIGGER OPENEBS IMAGE UPGRADE UTILS
+
+        - include_tasks: /common/utils/openebs/openebs_upgrade.yml
+          vars:
+            app_ns: "{{ namespace }}"
+            operator_ns: "{{ operator_namespace }}"
+            upgrade_duration: "{{ upgrade_time }}" 
```

Comment:
@vibhor995 is this upgrade_time is needed?

Reply:
@nsathyaseelan  not needed, just missed it to remove.